common-lib/src/main/java/com/google/cloud/tools/intellij/BasePluginInfoService.java

```diff
@@ -105,6 +105,19 @@ public boolean shouldEnable(Feature feature) {
     return false;
   }
 
+  @NotNull
+  @Override
+  public boolean isPluginInstalled(String pluginId) {
```

Comment:
Looking at this more carefully, I may have given you a bad suggestion (sorry):

These methods are not specific to a specific plugin as they are written. However, BasePluginInfoService gets instantiated for each of our plugins and so it doesn't make much sense to have this method here.

This is to check if the old JB plugin is enabled right? Could you just keep this check scoped to the old plugin check logic since its very specific? E.g. don't extract a helper and just use PluginManager#getPlugin(..) directly - its still only a couple lines of code.